versions.cfg

```diff
@@ -11,4 +11,4 @@ plone.recipe.codeanalysis = 2.0b1
 Sphinx = 1.3.1
 docutils = 0.12
 Pygments = 2.0
-six = 1.4
\ No newline at end of file
+six = 1.4
```

Comment:
Please keep this file as is. Cleanup should be done in seperate commits, though it's not necessary here.